test/unit/YawPitchControl/DeviceMotion.spec.js

```diff
@@ -100,6 +115,41 @@ describe("DeviceMotion", function() {
 			});
 		});
 
+		it("should not trigger devicemotion event on android", (done) => {
```

Comment:
@happyhj devicemotion is not triggered on android?

Following `inst.disable()` call is the reason for not firing event. Isn't it?

Reply:
I agree with the description is not clear.
I will update the test description like below

> should not trigger "devicemotion" event on android after calling disable()